util/libcrypto.num

```diff
@@ -5425,44 +5425,47 @@ ASN1_item_d2i_ex                        5552	3_0_0	EXIST::FUNCTION:
 ASN1_TIME_print_ex                      5553	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_get0_provider                  5554	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get0_provider              5555	3_0_0	EXIST::FUNCTION:
-OPENSSL_strcasecmp                      5556	3_0_3	EXIST::FUNCTION:
-OPENSSL_strncasecmp                     5557	3_0_3	EXIST::FUNCTION:
-X509_PUBKEY_set0_public_key             ?	3_2_0	EXIST::FUNCTION:
-OSSL_STACK_OF_X509_free                 ?	3_2_0	EXIST::FUNCTION:
-EVP_MD_CTX_dup                          ?	3_2_0	EXIST::FUNCTION:
-EVP_CIPHER_CTX_dup                      ?	3_2_0	EXIST::FUNCTION:
+ASYNC_set_mem_functions                 ?	3_2_0	EXIST::FUNCTION:
+ASYNC_get_mem_functions                 ?	3_2_0	EXIST::FUNCTION:
+BIO_recvmmsg                            ?	3_2_0	EXIST::FUNCTION:
+BIO_sendmmsg                            ?	3_2_0	EXIST::FUNCTION:
+BIO_s_dgram_mem                         ?	3_2_0	EXIST::FUNCTION:
+BIO_s_dgram_pair                        ?	3_2_0	EXIST::FUNCTION:DGRAM
+BIO_err_is_non_fatal                    ?	3_2_0	EXIST::FUNCTION:SOCK
+BIO_ADDR_dup                            ?	3_2_0	EXIST::FUNCTION:SOCK
+BIO_new_bio_dgram_pair                  ?	3_2_0	EXIST::FUNCTION:DGRAM
+BIO_meth_set_sendmmsg                   ?	3_2_0	EXIST::FUNCTION:
+BIO_meth_get_sendmmsg                   ?	3_2_0	EXIST::FUNCTION:
+BIO_meth_set_recvmmsg                   ?	3_2_0	EXIST::FUNCTION:
+BIO_meth_get_recvmmsg                   ?	3_2_0	EXIST::FUNCTION:
```

Comment:
Please revert all the changes in this file except for the adding the new function symbols from this PR.

Reply:
I thought make update generated that diff, but apparently it was a rebase artifact. Thanks! Reverted.